EDU-8: Describe subdirectory data links (#405)
seqeralabs/docs: platform_versioned_docs/version-24.2/data_studios/managing.mdx

> You can only delete a data studio when it's **stopped**. Select the three dots next to the status message and then select **Delete**. The data studio is deleted immediately and can't be recovered.

Reviewer: ```suggestion
You can only delete a Studio when it's **stopped**. Select the three dots next to the status message and then select **Delete**. The Studio is deleted immediately and can't be recovered.
```

> ## Limit Data Studio access to a specific cloud bucket subdirectory {#cloud-bucket-subdirectory}

Reviewer: ```suggestion
## Limit Studio access to a specific cloud bucket subdirectory {#cloud-bucket-subdirectory}
```

> For a cloud bucket that is writeable, as enabled by including the bucket in a compute environment's **Allowed S3 bucket** list, you can limit write access to that bucket from within a studio session.

Reviewer: ```suggestion
For a cloud bucket that is writeable, as enabled by including the bucket in a compute environment's **Allowed S3 bucket** list, you can limit write access to that bucket from within a Studio session.
```

To limit read-write access to a specific subdirectory, complete the following steps:

1. From your Seqera instance, select the **Data Explorer** tab.

> 1. Select Add Cloud Bucket.

Reviewer: ```suggestion
1. Select **Add Cloud Bucket**.
```

1. Complete the following fields:
   - **Provider**: Select your cloud provider.

>    - **Bucket path**: Enter the full path to the subdirectory of the bucket that you want to use with your Data Studio, such as `s3://1000genomes/data`.

Reviewer: ```suggestion
   - **Bucket path**: Enter the full path to the subdirectory of the bucket that you want to use with your Studio, such as `s3://1000genomes/data`.
```

   - **Name**: Enter a name for this cloud bucket, such as *1000-genomes-data-dir*, to indicate the bucket name and subdirectory path.
   - **Credentials**: Select your provider credentials.
   - Optional: **Description**: Enter a description for this cloud bucket.

> 1. Select **Add** to create the cloud bucket.

Reviewer 2: ```suggestion
1. Select **Add** to create a custom data-link to a subdirectory in the cloud bucket.
```

> When adding a Data Studio, when you configure **Mounted data**, you can select the cloud bucket that you created with the previous instructions.

Reviewer: ```suggestion
When you add a Studio and configure **Mounted data**, you can select the cloud bucket that you created with the previous instructions.
```

Reviewer 2: This should be:

```
When defining a new Studio session, you can configure the **Mounted data** by selecting the custom data-link 
 created by the previous steps.
```
